src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="set-field">
+        <legend>Personal information</legend>
+        <label
+          for="Surname"
+          class="form-field2"
+          tabindex="1">
+
+          Surname:
+          <input
+            id="Surname"
+            class="field"
+            type="text"
+            name="Surname"
+            required>
+        </label>
+
+        <label for="name" class="form-field">
+          Name:
+          <input
+            id="name"
+            class="field"
+            type="text"
+            name="name"
+            required>
```

Comment:
Fix it for all inputs.
```suggestion
          <input
            id="name"
            class="field"
            type="text"
            name="name"
            required
          >
```

Reply:
Готово